src/notifications/notifications.helpers.ts

```
@@ -0,0 +1,4 @@
+import Notification from './entities/notification.entity';
+export const notificationsEntities = [
+  Notification,
+];
```

ESLint

[error] Newline required at end of file but not found.

(eol-last)